src/linear_algebra/nonsingular_inverse.lean

```diff
@@ -57,14 +54,13 @@ section cramer
   After defining `cramer_map` and showing it is linear,
   we will restrict our proofs to using `cramer`.
 -/
-variables [comm_ring α] (A : matrix n n α) (b : n → α)
+variables (A : matrix n n α) (b : n → α)
 
 /--
   `cramer_map A b i` is the determinant of the matrix `A` with column `i` replaced with `b`,
   and thus `cramer_map A b` is the vector output by Cramer's rule on `A` and `b`.
 
-  If `A ⬝ x = b` has a unique solution in `x`, `cramer_map` sends a square matrix `A`
-  and vector `b` to the vector `x` such that `A ⬝ x = b`.
+  If `A ⬝ x = b` has a unique solution in `x`, `cramer_map Aᵀ` sends the vector `b` to `A.det • x`.
   Otherwise, the outcome of `cramer_map` is well-defined but not necessarily useful.
 -/
 def cramer_map (i : n) : α := (A.update_row i b).det
```

Comment:
I now see that though the comments refer to replacing a column, the implementation actually replaces a row. This is the reason we have a transpose in the statement of Cramer's rule.

I think this should be fixed but I have run out of minutes. I'll take this up again this evening if it is still pending.

Reply:
Ah yes, my old nemesis: rows versus columns. I'll push a commit that fixes it.

Reply:
@Vierkantor Didn't Oliver mean that we should update the code instead of the docstrings?

Reply:
Ah of course. I was thinking that the issue was the comments didn't get updated in #3393.

Reply:
I tried modifying `cramer_map`, but it causes too much breakage that I can't fix it right now :(

Reply:
@Vierkantor Thank you very much for attempting the modification! In the end, I didn't have time this evening but I definitely will tomorrow so I'll see if I can resolve everything then.

Reply:
I have added a commit which redefines `cramer_map` using `update_column` instead of `update_row` and allows me to state `cramers_rule` without having to insert a transpose, as:
```lean
  cramer A b = A.det • A⁻¹.mul_vec b :=
```

So I think this is fine now.